examples/IBFE/explicit/ex1/input2d.test

```diff
@@ -101,7 +103,8 @@ IBFEMethod {
    use_jump_conditions        = USE_JUMP_CONDITIONS
    use_consistent_mass_matrix = USE_CONSISTENT_MASS_MATRIX
    IB_point_density           = IB_POINT_DENSITY
-}
+   Phi_dt                     = DT
```

Comment:
The solvers need to be able to figure out the time step size internally. This should not be an input parameter.